unpackaged/functional_tests/address/classes/Address_FTST.cls

```diff
@@ -132,6 +132,26 @@ public with sharing class Address_FTST {
         return newContact;
     }
 
+    private static Contact createTestContactWithoutAddress(Id accountId) {
```

Comment:
Curious why not use util_testdata to create test contact? I thought we had a similar method.

Reply:
you're right, I'll reference the UTIL_UnitTestData_API.getMultipleTestContacts in my method and just add the extra field mappings that I need. Thanks!